kurtosis-devnet/pkg/build/docker.go

```diff
@@ -135,6 +178,16 @@ func WithDockerDryRun(dryRun bool) DockerBuilderOptions {
 	}
 }
 
+// WithDockerConcurrency sets the maximum number of concurrent builds.
+func WithDockerConcurrency(limit int64) DockerBuilderOptions {
+	if limit <= 0 {
+		limit = 1
+	}
```

Comment:
Nice. Should we also set an upper bound?

```suggestion
// WithDockerConcurrency sets the maximum number of concurrent builds.
func WithDockerConcurrency(limit int64) DockerBuilderOptions {
	if limit <= 0 {
		limit = 1
	}
	if limit >= 32 {
	    limit = 32
	}
```

Reply:
Also I wonder if the interface of `WithDockerConcurrency` would be nicer if it instead accepted a regular `int`.